src/airdrop/src/utils.rs

```diff
@@ -68,3 +68,8 @@ pub fn get_pre_codes(state: &mut State) -> CustomResult<Code> {
         None => Err(NoMoreCodes),
     }
 }
+
+/// Make the conversion to ERC-20 amount
+pub fn convert_to_erc20(amount: AirdropAmount) -> AirdropAmountERC20 {
+    AirdropAmountERC20(amount.0 as u128 * 10u128.pow(10))
```

Comment:
What does `amount.0` represent here exactly? So if it holds `1`, is it 1 ICP or 1 e8s?

Reply:
1 ICP - we set the  amount to be airdropped to people in ICP - only ninth last step do we convert to e8s.

Reply:
The only problem I see here still, is that you cannot distribute 1.5 ICP with this code and this contradicts to my understanding that we also want to airdrop fractional ICP amounts. I can't see how it would work with this code...